client/src/tests/Components/Tweet/UserName.test.js

```diff
@@ -0,0 +1,73 @@
+import UserName from "../../../components/Tweets/Tweet/UserName";
+import { useSelector } from "react-redux";
```

Comment:
Check if this can be deleted.

Reply:
When if this can be deleted error 'useSelector is not defined'

Reply:
If this will be deleted, there will be the error: ...
In case of deletion, there will be the error:  ...